doc/source/conf.py

```diff
@@ -69,9 +73,10 @@
               'matplotlib.sphinxext.plot_directive',
               ]
 
-# the following doesn't work with sphinx < 1.0, but will make a separate
-# sphinx-autogen run obsolete in the future
-#autosummary_generate = True
+# Autosummary always wants to use a `generated/` directory.
+# We generate with `make api-stamp`
+# This could change in the future
+autosummary_generate = False
```

Comment:
@matthew-brett Is the `make api-stamp` replaceable with some invocation of `autosummary`? I don't fully understand everything that's going on here.